torch/nn/_functions/dropout.py

```diff
@@ -66,3 +66,64 @@ def symbolic(g, input, p=0.5, train=False, inplace=False):
     def _make_noise(input):
         return input.new().resize_(input.size(0), input.size(1),
                                    *repeat(1, input.dim() - 2))
+
+
+class AlphaDropout(Dropout):
+
+    @staticmethod
+    def symbolic(g, input, p=0.5, train=False, inplace=False):
+        # See Note [Export inplace]
+        # NB: In inference mode, FeatureDropout is exported as an identity op.
+        from torch.onnx.symbolic import _unimplemented
+        if train:
+            return _unimplemented("AlphaDropout", "training mode")
+        return input
+
+    @classmethod
+    def forward(cls, ctx, input, p=0.5, train=False, inplace=False):
+        if p < 0 or p > 1:
+            raise ValueError("dropout probability has to be between 0 and 1, "
+                             "but got {}".format(p))
+        ctx.p = p
+        ctx.train = train
+        ctx.inplace = inplace
+
+        if ctx.p == 0 or not ctx.train:
+            return input
+
+        if ctx.inplace:
+            ctx.mark_dirty(input)
+            output = input
+        else:
+            output = input.clone()
+
+        ctx.noise = cls._make_noise(input)
+        if ctx.p == 1:
+            a = 0
+            b = ctx.noise
+        else:
+            ctx.noise.bernoulli_(1 - ctx.p)
+            a = ((3.09091329228798008 * ctx.p + 1) * (1 - ctx.p)) ** (-0.5)
```

Comment:
Can you comment that this is 1.7580993408473766 ** 2? (In fact, I'd feel better if we still had alpha written out explicitly as a variable.